src/app/services/exchanges/binance_exchange_service.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+from datetime import datetime
+import sqlalchemy_get_or_create
+
+from app.models import Candlestick, CurrencyPair, Exchange, Currency
+
+from app.services.exchanges.exchange_service_base import ExchangeServiceBase
+
+
+class BinanceExchangeService(ExchangeServiceBase):
+    EXCHANGE_CODE = "binance"
+
+    def add_exchange(self) -> Exchange:
+        (exchange, _) = sqlalchemy_get_or_create.update_or_create(
+            self.session,
+            Exchange,
+            code=self.EXCHANGE_CODE,
+            defaults={"name": "Binance Exchange"},
```

Comment:
Suggestion: following the `EXCHANGE_CODE` design, wouldn't this fit an `EXCHANGE_NAME` one?

Reply:
Done